src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -196,17 +197,56 @@
 }
 
 
+@click.command()
+@click.argument("raw_output_dir", type=click.Path(exists=True))
+@click.argument("final_output_dir", type=click.Path(exists=True))
+@click.option("-v", "verbose", count=True, help="Configure logging verbosity.")
+@click.option(
+    "--pdb",
+    "with_debugger",
+    is_flag=True,
+    help="Drop into python debugger if an error occurs.",
+)
+@click.option("-b", "--mark-best", is_flag=True)
+@click.option(
+    "-t",
+    "--test-run",
+    is_flag=True,
+)
+@click.option(
+    "-a", "--artifact-path", type=click.Path(exists=True), default=paths.DEFAULT_ARTIFACT
+)
 def build_results(
```

Comment:
Remove cli wrappipng and have workflow call `make_results`

Reply:
This didn't work b/c make_results requires just output_dir but at this point we've already split that into raw_dir and final_dir.